docs/api.md

```diff
@@ -765,6 +767,60 @@ kinto.events.on("deprecated", function(event) {
 });
 ```
 
+### `Collection` instance
+
+#### Change event
+
+Triggered when a successful operation was executed. Only one is sent per transaction.
+
+The `event` argument contains the following information:
+
+- `targets`: a list of mappings with an `action` attribute and the other attributes of the respective event as described below.
+
+```js
+const kinto = new Kinto();
+const articles = kinto.collection("articles");
+
+articles.events.on("change", function(event) {
+  const first = events.targets[0];
+  console.log(first.action);
+  console.log(first.data);
+});
+
+articles.delete(recordId);
+```
+
+#### Per action event
+
+Atomic events are sent for ``create``, ``update`` and ``delete``.
+
+Every event contains the following attributes:
+
+- ``data``: the record that was created, updated or deleted
+
+The ``update`` event contains an additionnal attribute:
```

Comment:
OK, I see that there's no special field for `id`, but you can retrieve them from the records provided as `data` and `oldRecord`.